src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/checkout_payment_form.html

```diff
@@ -2,33 +2,41 @@
 {% load ibanformat %}
 {% load bootstrap3 %}
 
-<p>{% blocktrans trimmed %}
-    After completing your purchase, we will ask you to transfer the money to the following
-    bank account, using a personal reference code:
-{% endblocktrans %}</p>
+{% if details or code %}
+    <p>{% blocktrans trimmed %}
+        After completing your purchase, we will ask you to transfer the money to the following
+        bank account, using a personal reference code:
+    {% endblocktrans %}</p>
 
-{% if settings.bank_details_type == "sepa" %}
-    <dl class="dl-horizontal">
-        <dt>{% trans "Account holder" %}: </dt><dd>{{ settings.bank_details_sepa_name }}</dd>
-        <dt>{% trans "IBAN" %}: </dt><dd>{{ settings.bank_details_sepa_iban|ibanformat }}</dd>
-        <dt>{% trans "BIC" %}: </dt><dd>{{ settings.bank_details_sepa_bic }}</dd>
-        <dt>{% trans "Bank" %}: </dt><dd>{{ settings.bank_details_sepa_bank }}</dd>
-    </dl>
-{% endif %}
+    {% if not code %}<div class="user-select-none">{% endif %}
+    {% if settings.bank_details_type == "sepa" %}
+        <dl class="dl-horizontal" data-toggle="tooltip" title="{% trans "Please do not yet start a payment. We'll assign you a personal reference code after you completed the order." %}">
```

Comment:
The tooltip should be handled the same as in the other template:

```suggestion
    {% if not code %}<div class="user-select-none" data-toggle="tooltip" title="{% trans "Please do not yet start a payment. We'll assign you a personal reference code after you completed the order." %}">{% endif %}
    {% if settings.bank_details_type == "sepa" %}
        <dl class="dl-horizontal">
```